api/services/orders/orders_store.go

```diff
@@ -0,0 +1,46 @@
+package orders
+
+import (
+	"database/sql"
+	"log"
+	"github.com/sockify/sockify/types"
+)
+
+type OrderStore struct {
+	db *sql.DB
+}
+
+func NewOrderStore(db *sql.DB) *OrderStore {
+	return &OrderStore{db: db}
+}
+
+func (s *OrderStore) OrderExistsByID(orderID int) (bool, error) {
```

Comment:
nit: I don't see it used anywhere right now. Are you planning to check that the order exists before updating?

Reply:
sounds good